auth0/resource_auth0_user.go

```diff
@@ -150,7 +156,7 @@ func buildUser(d *schema.ResourceData) *management.User {
 		}
 	}
 
-	if u.Username != nil || u.Password != nil || u.EmailVerified != nil || u.PhoneVerified != nil {
+	if u.Username != nil || u.Password != nil || u.EmailVerified != nil || u.PhoneVerified != nil || u.Nickname != nil {
 		// When updating email_verified, phone_verified, username or password
```

Comment:
The comment talk about email_verified, phone_verified, username or password.
Are you sure the Nickname have the same behavior ?

If so, I suggest to update the comment to `email_verified, phone_verified, username, nickname or password`

Reply:
well spotted! I'll revert that line and retest.

Reply:
reverted, btw